packages/gatsby-theme-apollo-core/package.json

```diff
@@ -13,12 +13,13 @@
   "dependencies": {
     "@apollo/space-kit": "^5.6.0",
     "@emotion/core": "^10.0.7",
-    "@emotion/styled": "^10.0.7",
-    "@svgr/webpack": "^4.2.0",
-    "gatsby-plugin-emotion": "^4.0.3",
-    "gatsby-plugin-less": "^3.0.1",
-    "gatsby-plugin-react-helmet": "^3.0.2",
-    "gatsby-plugin-svgr": "^2.0.1",
+    "@emotion/react": "^11.0.0",
+    "@emotion/styled": "^11.0.0",
```

Comment:
The update for Emotion 11 and accompanying Gatsby plugin also require `@emotion/styled`. Not sure if we still need `@emotion/core` or if it's safe to remove.

Reply:
Update: the version of space-kit that we're using stil lrelies on @emotion/core, so I'm leaving it in